src/Auth/DoctrineUserProvider.php

```diff
@@ -128,6 +129,8 @@ protected function getRepository()
      */
     protected function getEntity()
     {
-        return new $this->entity;
```

Comment:
That will give the same error when having forced parameters in your constructor.